iocage/cli/__init__.py

```diff
@@ -25,7 +25,7 @@
 import os
 import re
 import signal
-import subprocess as su
+import subprocess  # nosec: B804
```

Comment:
this should be B404, shouldn't it?
i disabled that entirely in the makefile, and tried disabling it in setup.cfg

Reply:
aye! will update

Reply:
Just figured out that despite of a defined exception, `# nosec: <ANYTHING>` does skip all bandit warnings for this line. We should be careful with that and fix upstream